automated/health-checks/tests/health-check.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+TEST_CASE_NAME=$(basename "$0" .sh)
+
+# Check CPU load
+cpu_load=$(uptime | awk -F 'load average:' '{print $2}' | awk -F, '{print $1}' | awk '{$1=$1};1')
+echo "CPU Load: $cpu_load"
+
+# Check available memory
+free_mem=$(free -m | awk 'NR==2{printf "%.2f%%", $3*100/$2}')
+echo "Available Memory: $free_mem"
+
+# Check disk space
+disk_space=$(df -h / | awk 'NR==2{printf "%s", $5}')
+echo "Disk Space: $disk_space"
+
+lava-test-case "$TEST_CASE_NAME-Health-Check" --result pass
```

Comment:
What's the purpose of a check script which always return successful.also the cpu load and free memory is dumped but never checked. Am I missing something?

Reply:
The purpose of the health check is making sure that the device is healthy to run jobs on. We agreed that "healthy" means an SSH connection can be established. If this is the case then the health check will pass. If no SSH connection can be established the device will be taken offline.
Thus, the health check returns successful only if an SSH connection can be established.

It's debatable what "healthy" means exactly. It could also mean that the device is flashable, since we might want to flash the devices every time before executing tests on them. But we're not sure how one would check for that so we agreed on this.